inference-engine/src/transformations/src/transformations/convert_extract_image_patches_to_reorg_yolo.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/convert_extract_image_patches_to_reorg_yolo.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset3.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+ngraph::pass::ConvertExtractImagePatchesToReorgYolo::ConvertExtractImagePatchesToReorgYolo() {
+    auto image = std::make_shared<ngraph::pattern::op::Label>(ngraph::element::f32, ngraph::Shape{1, 1, 1, 1});
+    auto eip = std::make_shared<ngraph::opset3::ExtractImagePatches>(image, ngraph::Shape{1, 1}, ngraph::Strides{1, 1}, ngraph::Shape{1, 1},
+            ngraph::op::PadType::VALID);
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher &m) {
+        auto &pattern_to_output = m.get_pattern_value_map();
+        auto extract_image_patches =  std::dynamic_pointer_cast<ngraph::opset3::ExtractImagePatches>(m.get_match_root());
+
+        if (!extract_image_patches) {
```

Comment:
You have added a check for the EIP to the transformation callback but you don't run the transformation callback here. So the EIP will be converted to ReorgYolo always. Please, refer to the file inference-engine/src/transformations/src/transformations/reduce_l1_decomposition.cpp in the review https://github.com/openvinotoolkit/openvino/pull/1799/files

Reply:
Please, provide an example of benchmark_app run with `-pc` command line to be sure that the EIP has not been unfolded into primitives.

Reply:
`m_transformation_callback` check added.